beacon-chain/blockchain/state_balance_cache_test.go

```diff
@@ -34,7 +34,8 @@ func testStateFixture(opts ...testStateOpt) state.BeaconState {
 	for _, o := range opts {
 		o(a)
 	}
-	s, _ := state_native.InitializeFromProtoUnsafeAltair(a)
+	s, err := state_native.InitializeFromProtoUnsafeAltair(a)
+	_ = err
```

Comment:
Is there a reason why we are silencing the error rather than having a: `require.NoError(t, err)`

Reply:
there's no `t`,  I didn't want to touch the existing code as much as possible.